__tests__/app.test.js

```diff
@@ -85,3 +86,33 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+//5
+describe("/api/articles", () => {
+  test("GET - status 200 - responds list of articles sorted date descending", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then((response) => {
+        const { allArticles } = response.body;
+
+        expect(allArticles.length > 0).toBe(true);
```

Comment:
I'd encourage you to refactor this and add an actual .length test which says how many items are in the array. Currently it just says there's more than 0, which isn't all that informative. Have the actual amount here instead :)